app/main.py

```diff
@@ -1,3 +1,30 @@
 def create_dict(*args):
-    # write your code here
-    pass
+    res_dict = {}
+    is_not_tuple = True
+
+    for iterator, value in enumerate(args):
+
+        if (
+            isinstance(value, list) or
+            isinstance(value, set) or
+            isinstance(value, dict)
```

Comment:
"isinstance" can take tuple as second argument, make this conditions shorter

Reply:
Can u explain a little bit more deployed?